output pure values from proc stat file

Keeo opened the pull request:
How does it look to you? @jrgp 
We need review and complete these points before merging.

- [x] option field name and entry function name
- [ ] data structure
- [ ] documentation
- [x] tests
- [x] is html output required?

How does `cpuStats` look to you? I don't think it's well suited name, so your input here would be helpful.

Then there is how to structure data. Right now the output is:
```javascript
[{'name':'overall','row':'543 543'},{'name':'1','row':'432 432'}]
```

And html output. Is it required? This output is designed to be used in programs and not for human eyes.

I tried mimic your coding style as much as possible.
fixes #15

jrgp commented:
Hi Keeo! Thanks for your contribution to Linfo.

I agree that cpuStats is a really vague name. How about kernelCpuStats? These values are extremely raw, and including "kernel" in the name probably makes it sound like they're less human-readable than you'd expect.

Keeo commented:
New update:
- added test
- removed anonymous function *(to support declared php version)*
- removed `[]` array initialization *(to support declared php version)*
- renamed fields to `kernelCpuStats`

Does data structure looks good to you and are we ready to write documentation?
